book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ]  This method is meant to  create a new `Rental` object, passing `date`,  and `person` as arguments. This method should to be more involved in creating a new rental instance rather than just adding an existing rental to a collection.

- Considering the fact we're trying to create a rental and associate it with a person and date, kindly edit the code snippet to the  appropriate pattern to achieve the  `has-many' association.`

### Sample:
```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```